plugins/destination/clickhouse/queries/test_data/col_add.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `table_name` ADD COLUMN `my_col` Int64
```

Comment:
Let's rename this directory to the more standard `testdata` (no underscore) so that it gets ignored by the `go` tool.